guides/releasing/Release Workflow.md

```diff
@@ -0,0 +1,328 @@
+    - [4.4. Submitting to the stores](#44-submitting-to-the-stores)
+  - [Stage 5 - ExpoKit and standalone apps](#stage-5---expokit-and-standalone-apps)
+    - [5.1. Updating ExpoKit](#51-updating-expokit)
+    - [5.2. Make shell app build](#52-make-shell-app-build)
+    - [5.3. Make adhoc client shell app](#53-make-adhoc-client-shell-app)
+    - [5.4. Deploy Turtle with new shell tarballs](#54-deploy-turtle-with-new-shell-tarballs)
+  - [Stage 6 - Final release](#stage-6---final-release)
+    - [6.1. Publishing final packages to NPM registry](#61-publishing-final-packages-to-npm-registry)
+    - [6.2. Publishing final project templates](#62-publishing-final-project-templates)
+  - [Stage 7 - Snack](#stage-7---snack)
+    - [7.1. Add SDK support to Snack](#71-add-sdk-support-to-snack)
+  - [Stage 8 - Press release](#stage-8---press-release)
+    - [8.1. Publish a blog post](#81-publish-a-blog-post)
+    - [8.2. Post on Twitter](#82-post-on-twitter)
+
+# Stage 1 - Versioning
+
+## 1.1. Dropping old SDKs
+
+**Why:** We tend to support old SDK versions up to 6 months since they were released. Once we release a new one, it's a good opportunity to drop some old ones that are already older than 6 months.
+
+**How:**
+
+- Create a new branch from `master`.
+- **iOS**:
+  - Run `et remove-sdk --platform ios` to remove the oldest SDK version from iOS codebase and regenerate Pods.
+  - Make sure Expo client builds in Xcode as expected.
+- **Android**:
+  - Run `et remove-sdk --platform android` to remove Android's `expoview-abiXX_0_0` library and versioned code.
+  - This script will print all references to the SDK version it just removed - check all of them and remove them if possible (remove legacy code).
+  - Make sure Expo client builds in Android Studio as expected.
+- Repeat **iOS** and **Android** specific steps if you want to delete more SDK versions.
+- Commit changes and create a pull request to `master` branch.
+
+## 1.2. Update React Native
+
+**Why:** Each SDK version has its own version of React Native. If we're planning to update React Native version for the upcoming SDK then we need to update our fork of React Native and update `react-native-lab/react-native` submodule in this repo which is the source of truth for `react-native` version used throughout the repository.
+    
+**How:**
+
+- Go to `react-native-lab/react-native` submodule.
+- Coordinate with James (@ide) to create a new branch for new SDK in our `react-native` fork (`sdk-XX` typically, where `XX` is the major number of the SDK version).
+- Run `et update-react-native`. This expotools command copies `ReactAndroid` and `ReactCommon` folders from the submodule to the respective paths: `android/ReactAndroid` and `android/ReactCommon` and then executes `ReactAndroidCodeTransformer` that applies some Expo-specific code transformations.
+- Add your git changes from both `react-native-lab` and `android` folders and create a pull request to `master` branch.
+
```

Comment:
There should be a step here for unversioned QA! This is especially important on Android because we build ReactAndroid into an aar in the versioning step, so if there are any issues that are discovered after versioning you have to redo the whole versioning step (which takes quite a while 😞 ). So we should have a step here for QA'ing at least the RN dev tools on Android. I usually QA everything else as well, because then if there are any issues I can fix them right away and know that any issues I find in QA after versioning must have to do with versioning itself.